lib/local_process.rb

```diff
@@ -11,13 +11,14 @@ class LocalProcess
 
     DEFAULT_TIMEOUT = 3600
     CENSOR_CMDS = [
-      'oc exec .*bearer',   # oc exec elasticsearch-* -- curl "Authorization: Bearer ***"
-      'oc rsh .*bearer',    # oc rsh elasticsearch-* -- curl "Authorization: Bearer ***"
-      'oc login .*--token', # oc login --token=***
-                            # oc login --token ***
-      'oc login .*-p',      # oc login --passwordr= ***
-                            # oc login --password ***
-                            # oc login -p ***
+      'oc exec .*bearer',    # oc exec elasticsearch-* -- curl "Authorization: Bearer ***"
+      'oc exec .*--dcreds',  # oc exec -- skopeo copy --dcreds username:****
```

Comment:
Searching the logs, i got,
```
skopeo --debug --insecure-policy copy --dest-tls-verify\=false --dcreds ***
skopeo inspect --tls-verify\=false --creds ***
```
Perhaps use `oc exec .*--d?creds` to cover both of them ?

Reply:
@jmekkatt Just like liangxia mentioned, we have other places where use '-creds', 

```
$ grep -nrE '\-\-\d?creds'
verification-tests/features/online/imagestreams.feature:289:      | --dcreds                                                    |
verification-tests/features/build/buildlogic.feature:525:      | --creds                                                                    |
verification-tests/features/build/buildlogic.feature:542:      | --creds                                           |
verification-tests/features/registry/remote.feature:29:      | --creds                    |
verification-tests/features/registry/remote.feature:39:      | --dcreds                   |
verification-tests/features/registry/remote.feature:57:      | --creds                    |
cucushift/online/imagestreams.feature:68:      | --dcreds                                                             |
cucushift/online/imagestreams.feature:97:      | --dcreds                                                                |
cucushift/online/imagestreams.feature:115:      | --dcreds                                                                |
cucushift/online/imagestreams.feature:144:      | --dcreds                                                                |
cucushift/build/buildlogic.feature:87:      | --creds            |
cucushift/build/buildlogic.feature:363:      | --creds            |
cucushift/build/buildlogic.feature:395:      | --creds            |
cucushift/cli/limits.feature:196:      | --dcreds                                  |
cucushift/admin/quota.feature:744:      | --dcreds                                                                 |
cucushift/registry/registry.feature:53:      | --dcreds                   |
cucushift/registry/remote.feature:26:      | --dcreds                             |
cucushift/registry/remote.feature:45:      | --dcreds                             |
cucushift/registry/remote.feature:159:      | --dcreds                             |

```

Reply:
one more tip, give the test results pipe output `| sed 's/\[[0-9;]*[mGKF]//g' | rhpaste` to remove ANSI-colors tags. looks better.